src/helpers/util.ts

```diff
@@ -207,3 +208,15 @@ export function isValidOracleType(
 ): oracleType is OracleType {
   return !!oracleType && oracleType in oracleTypes;
 }
+
+export function isWagmiAddress(
+  maybeAddress: string | undefined,
+): maybeAddress is Address {
+  if (!maybeAddress) return false;
+  return "0x" == maybeAddress.slice(0, 2);
+}
+export function assertWagmiAddress(maybeAddress: string): Address {
```

Comment:
no need to change anything but you might find this cool https://blog.logrocket.com/assertion-functions-typescript/

you can annotate this type of function like `asserts value is Address` and it will automatically treat the value as `Address` after. saves some typecasting